File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/HomeServicesWebhostingLinks.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Map of webhosting service IDs to their hostnames for the account dashboard.
 */
class HomeServicesWebhostingLinks
{
    /**
     * Link to a website.
     */
    #[DTA\Data(field: "376359", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $_376359 = null;

    /**
     * Link to a website.
     */
    #[DTA\Data(field: "376473", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $_376473 = null;

    /**
     * Link to a website.
     */
    #[DTA\Data(field: "386218", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $_386218 = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/HomeServicesWebhostingLinks.scala
package io.swagger.server.model


/**
 * Map of webhosting service IDs to their hostnames for the account dashboard.
 *
 * @param 376359 Link to a website. for example: ''Doefamily.com''
 * @param 376473 Link to a website. for example: ''fancytush.com''
 * @param 386218 Link to a website. for example: ''admincoded.net''
 */
case class HomeServicesWebhostingLinks (
  376359: Option[String],
  376473: Option[String],
  386218: Option[String]
)

